src/core_plugins/kbn_vislib_vis_types/public/controls/point_series/category_axis.html

```diff
@@ -43,46 +51,57 @@
         ng-class="{ 'fa-caret-down': isCategoryAxisAdvancedOptionsOpen, 'fa-caret-right': !isCategoryAxisAdvancedOptionsOpen }"
         class="kuiIcon fa-caret-right kuiSideBarOptionsLink__caret"
       ></span>
-      <span class="kuiSideBarOptionsLink__text">
-        <span ng-show="!isCategoryAxisAdvancedOptionsOpen">
-          Show
-        </span>
-        <span ng-show="isCategoryAxisAdvancedOptionsOpen">
-          Hide
-        </span>
-        Advanced Options
-      </span>
+      <span
```

Comment:
Especially with i18n, that doesn't look like a good solution anymore, neither code wise nor for translators later. Let's instead change this to just two separate labels (`Show Advanced Options` and `Hide Advanced Options`), which will just use both `ng-show` directly.